File: kopi.js
const readline = require('readline');

function hitungKopi(jumlahGelas) {
    let total = jumlahGelas * 5000;
    if (jumlahGelas > 3) {
        total -= 2000;
    }
    return total;
}

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

console.log("=== PROGRAM HITUNG KOPI ===");
rl.question("Masukkan jumlah gelas kopi yang dibeli: ", (inputGelas) => {
    const gelas = parseInt(inputGelas);
    const totalBayar = hitungKopi(gelas);
    
    console.log(`Total harga yang harus dibayar       : Rp ${totalBayar}`);
    rl.close();
});

File: kopi.py
def hitungKopi(jumlahGelas):
    total = jumlahGelas * 5000
    if jumlahGelas > 3:
        total -= 2000
    return total

if __name__ == "__main__":
    print("=== PROGRAM HITUNG KOPI ===")
    gelas = int(input("Masukkan jumlah gelas kopi yang dibeli: "))
    
    total_bayar = hitungKopi(gelas)
    print(f"Total harga yang harus dibayar       : Rp {total_bayar}")
